types/ember-cli-fastboot/test/ember-cli-fastboot-tests.ts

```diff
@@ -0,0 +1,23 @@
+import { assertType } from './assert';
+import { FastBoot as FastBootService, FastbootRequest, Shoebox } from 'ember-cli-fastboot/services/fastboot';
```

Comment:
With the fix for the export location:

```suggestion
import FastBootService, { FastbootRequest, Shoebox } from 'ember-cli-fastboot/services/fastboot';
```